scripts/calendar/hindu_generator.py

```
@@ -0,0 +1,876 @@
+        while dt <= ashwin_ama + timedelta(days=15):
+            ap = self._aparahna(dt)
+            ap_prev = self._aparahna(dt - timedelta(days=1))
+            t = self._tithi(ap)
+            t_prev = self._tithi(ap_prev)
+
+            # If Navami present at Aparahna -> first occurrence
+            if t == 9:
+                return dt
+
+            # If Navami skipped between Aparahnas (8->10)
+            elif t == 10 and t_prev == 8:
```

Ruff (0.15.12)

[warning] Unnecessary `elif` after `return` statement

Remove unnecessary `elif`

(RET505)